handbook/company/pricing-features-table.yml

```diff
@@ -299,7 +299,8 @@
     #  ╠╦╝║╣ ║║║║ ║ ║ ║╣   ╚═╗║╣  ║  ║ ║║║║║ ╦╚═╗
     #  ╩╚═╚═╝╩ ╩╚═╝ ╩ ╚═╝  ╚═╝╚═╝ ╩  ╩ ╩╝╚╝╚═╝╚═╝
     - industryName: Remote settings
-      description: Configure agent options remotely, over the air.  (Includes osquery config, fleetd options, and osquery startup flags.)
+      waysToUse:
```

Comment:
By convention, there is a certain order to the keys in this file.

Reply:
What's the convention? I made some changes and I assumed it's `description` after names and `waysToUse` last.

Reply:
@noahtalerman "documentationUrl" and "description" are nested under "ways to use".